pkg/executor/statefulset.go

```diff
@@ -65,6 +65,11 @@ func UpdateStatefulSetReplicas(
 	if err == nil {
 		return nil
 	}
+
+	// See GitHub issue #194: Migrate Client().Update() calls
+	// back to Patch() calls.
+	// https://github.com/bluek8s/kubedirector/issues/194
```

Comment:
We also need a comment like this in executor.UpdatePodService.

Reply:
Done.